docs/dev.rst

```diff
@@ -238,10 +238,12 @@ Then, you should run the following commands::
     # Rebuild static assets
     openedx-assets build --env=dev
 
-After running all these commands, your edx-platform repository will be ready for local development. To debug a local edx-platform repository, you can then add a ``import ipdb; ipdb.set_trace()`` breakpoint anywhere in your code and run::
+After running all these commands, your edx-platform repository will be ready for local development. To debug a local edx-platform repository, you can then add a `python breakpoint <https://docs.python.org/3/library/functions.html#breakpoint>`__ with ``breakpoint()`` anywhere in your code and run::
 
     tutor dev start --mount=/path/to/edx-platform lms
 
+``PYTHONBREAKPOINT`` can be modified in the `config.yml` in order to use a custom debugger. The default debugger is ``pdb.set_trace``.
```

Comment:
This should be modified to reflect that the default debugger can be modified by setting an environment variable in the Docker image.

Reply:
This line has been modified to be more accurate.